build.gradle

```diff
@@ -61,9 +61,10 @@ buildscript {
             "jsonAssert"                  : "org.skyscreamer:jsonassert:1.4.0",
             "striktMockk"                 : "io.strikt:strikt-mockk:0.30.1",
             "striktCore"                  : "io.strikt:strikt-core:0.30.1",
+            "jsonMock"                    : "org.json:json:20211205",
 
-            // PayPal
-            "paypalNativeCheckout"        : "com.paypal.checkout:android-sdk:0.5.2"
+            // Browser Switch
+            "browserSwitch"               : "com.braintreepayments.api:browser-switch:2.1.1"
```

Comment:
nit: can we move this up above the test dependencies?

Reply:
done 👍